app/models/support_interface/providers_filter.rb

```diff
@@ -105,6 +112,17 @@ def filter_records(providers)
         providers = providers.left_outer_joins(:provider_permissions).where(provider_users_providers: { id: nil })
       end
 
+      if applied_filters[:accredited_provider].present?
```

Comment:
LGTM but conscious `#filter_records` is getting quite large - could be worth refactoring at some point